website/src/index.css

```diff
@@ -151,3 +151,8 @@ body {
     hsla(203, 11%, 15%, 0.4);
   color: #aaa !important;
 }
+.bg-clip-text {
+  background: -webkit-linear-gradient(#ff8533, #ff4da6);
+  background-clip: border-box;
+  -webkit-text-fill-color: transparent;
+}
```

Comment:
Can you change the class these properties are added to to a unique class? We don't want these rules to be applied to all future uses of `.bg-clip-text`